I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 8, in script
    play music ecoplay-bgs
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
  File "renpy/common/000statements.rpy", line 32, in _audio_eval
    return eval(expr, locals=store.audio.__dict__)
  File "game/script.rpy", line 8, in <module>
    play music ecoplay-bgs
NameError: name 'ecoplay' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 8, in script
    play music ecoplay-bgs
  File "C:\Users\ibrah\OneDrive\Documents\A game renpy\renpy-8.1.3-sdk\renpy\ast.py", line 2259, in execute
    self.call("execute")
  File "C:\Users\ibrah\OneDrive\Documents\A game renpy\renpy-8.1.3-sdk\renpy\ast.py", line 2241, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\ibrah\OneDrive\Documents\A game renpy\renpy-8.1.3-sdk\renpy\statements.py", line 342, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
  File "renpy/common/000statements.rpy", line 32, in _audio_eval
    return eval(expr, locals=store.audio.__dict__)
  File "C:\Users\ibrah\OneDrive\Documents\A game renpy\renpy-8.1.3-sdk\renpy\python.py", line 1153, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\ibrah\OneDrive\Documents\A game renpy\renpy-8.1.3-sdk\renpy\python.py", line 1146, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 8, in <module>
    play music ecoplay-bgs
NameError: name 'ecoplay' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.3.23091805
GreeniMe 1.0
Fri May 31 16:54:14 2024
